docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     profiles:
       - pmm
     image: ${PMM_CONTAINER:-perconalab/pmm-server:dev-container}
+    restart: always
```

Comment:
Not sure I understood the question, but I removed that line.

Reply:
I'm running my dev env on a spot instance which may occasionally get stopped or even torn down. This is why I have to add `restart: always`, otherwise the devcontainer doesn't start automatically.

Reply:
Yes, I have no problems to have it. I was just interested why do you need it. Let's keep it if it helps you.